src/utils.ts

```diff
@@ -255,6 +291,31 @@ export const assetFromJSON = (asset: any): OpenSeaAsset => {
   return fromJSON
 }
 
+export const transactionFromJSON = (transaction: any): Transaction => {
+  return {
+    id: transaction.id,
+    fromAccount: accountFromJSON(transaction.from_account),
+    toAccount: accountFromJSON(transaction.to_account),
+    createdDate: transaction.created_date,
```

Comment:
we've been converting these to javascript Dates in other places, so let's do that here too? `new Date(`${transaction.created_date}Z`)` and same for modifiedDate

Reply:
👍